Userland/Applications/FileManager/main.cpp

```diff
@@ -1074,18 +1077,18 @@ int run_in_windowed_mode(RefPtr<Core::ConfigFile> config, String initial_locatio
         directory_view.delete_action().set_enabled(!tree_view.selection().is_empty());
     };
 
-    tree_view.on_focus_change = [&]([[maybe_unused]] const bool has_focus, [[maybe_unused]] const GUI::FocusSource source) {
+    tree_view.on_focus_change = [&]([[maybe_unused]] bool const has_focus, [[maybe_unused]] GUI::FocusSource const source) {
```

Comment:
I don't think the `bool` needs `const`, and the `[[maybe_unused]]` could be removed, I think.

Reply:
Sure.